### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by sonalkr132** [openSUSE/osem#957]
#### formtastic
Visiting the questions form gives:
> The :member_label option is deprecated in favour of `passing an Array of label/value pairs like [['Justin', 2], ['Kate', 3]] into :collection directly (consider building the array in your model using Model.pluck)`. :member_label will be removed in the next version. (called from block in _app_views_conference_registrations__questions_html_haml___3312550169329296742_52833760 at /home/user/osem/app/views/conference_registrations/_questions.html.haml:2)

Source: https://github.com/openSUSE/osem/blob/master/app%2Fviews%2Fconference_registrations%2F_questions.html.haml#L5

#### transitions (May be?)
On update of state of proposal, we get:
> `serialized_attributes` is deprecated without replacement, and will be removed in Rails 5.0. (called from withdraw at /home/user/osem/app/controllers/proposal_controller.rb:93

Weird thing is that transitions gem is not using `serialized_attributes` method in its codebase. Could be one of its dependency.

#### htmlentities
Running tests gives this:
> /home/user/.rvm/gems/ruby-2.3.0/gems/htmlentities-4.3.1/lib/htmlentities/mappings/expanded.rb:465: warning: key "inodot" is duplicated and overwritten on line 466

[See on travis](https://travis-ci.org/openSUSE/osem/builds/122555128#L1727).

Seems to be fixed on updated version of the gem in interest.

**Comment by nishanthvijayan** [comment]
serialized_attributes deprecated warning is being triggered by paper_trail
```
DEPRECATION WARNING: `serialized_attributes` is deprecated without replacement, and will be removed in Rails 5.0. (called from restart at /vagrant/app/controllers/proposal_controller.rb:139)
/usr/lib64/ruby/gems/2.1.0/gems/activerecord-4.2.5.2/lib/active_record/attribute_methods/serialization.rb:56:in `serialized_attributes'
  /usr/lib64/ruby/gems/2.1.0/gems/paper_trail-3.0.1/lib/paper_trail/has_paper_trail.rb:112:in `serialize_attributes_for_paper_trail'
  /usr/lib64/ruby/gems/2.1.0/gems/paper_trail-3.0.1/lib/paper_trail/has_paper_trail.rb:339:in `block in object_attrs_for_paper_trail'
  /usr/lib64/ruby/gems/2.1.0/gems/paper_trail-3.0.1/lib/paper_trail/has_paper_trail.rb:338:in `tap'
  /usr/lib64/ruby/gems/2.1.0/gems/paper_trail-3.0.1/lib/paper_trail/has_paper_trail.rb:338:in `object_attrs_for_paper_trail'
  /usr/lib64/ruby/gems/2.1.0/gems/paper_trail-3.0.1/lib/paper_trail/has_paper_trail.rb:268:in `record_update'
```
[has_paper_trail.rb:112](https://github.com/airblade/paper_trail/blob/v3.0.1/lib/paper_trail/has_paper_trail.rb#L112)
This was [fixed in v 4.1.0] (https://github.com/airblade/paper_trail/commit/c6ffd603e40c9cd049e42639e274f92fa66a88d8).We use v3.0.1  
  
  
The htmlentities related warning was [fixed](https://github.com/threedaymonk/htmlentities/commit/cd69ae59789fd8f435a2a1ba3a18d65f0c496b74#diff-04565388361dacc058741050a801d885) in v4.3.3.We currently use v4.3.1

**Comment by Ana06** [comment]
I think that this issue can be closed. :smiley:
